assignment02a/src/main/java/com/codenation/assingment02/SmartHome.java

```diff
@@ -0,0 +1,106 @@
+package com.codenation.assingment02;
+
+import java.util.*;
+
+public class SmartHome {
+    public static void main(String args[])
+    {
+        int hrsInADay = 2;
+        
+        HashMap<Integer, ArrayList<Event>> hrEventsMap = new HashMap<Integer, ArrayList<Event>>();
+        HashMap<String, Appliance> nameApplianceMap = new HashMap<String, Appliance>();
+        
+        System.out.println("Enter number of appliances");
+        
+        Scanner in = new Scanner(System.in);
+        
+        int numDevices = in.nextInt();
+        
+        
+        for(int i = 0; i < numDevices; i++)
```

Comment:
take input from file and preferably in a function other than main, easier to test